gqcp/include/Operator/SecondQuantized/SQTwoElectronOperator.hpp

```diff
@@ -416,7 +416,7 @@ auto operator*(const Scalar& scalar, const SQTwoElectronOperator<OperatorScalar,
 
     auto G = op.allParameters();
     for (size_t i = 0; i < Components; i++) {
-        G[i] *= scalar;
+        G[i] = scalar * G[i];
```

Comment:
```suggestion
        G[i] *= scalar;
```

There isn't really a reason to change this implementation.

Reply:
Actually, I changed it because the test gave an error with the initial implementation, saying the operator required different arguments.

Reply:
Oh, really? I didn't know. Thanks for pointing that out.